pkg/yurthub/proxy/proxy.go

```diff
@@ -98,7 +102,20 @@ func (p *yurtReverseProxy) buildHandlerChain(handler http.Handler) http.Handler
 }
 
 func (p *yurtReverseProxy) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
-	if !hubutil.IsKubeletLeaseReq(req) && p.loadBalancer.IsHealthy() {
+	now := time.Now()
+	isKubeletLeaseReq := hubutil.IsKubeletLeaseReq(req)
+	if isKubeletLeaseReq {
+		p.loadBalancer.Resume()
+		p.lastKubeletLeaseReqTime = now
+	}
```

Comment:
Please use `else if` here.
Also please combine three conditions together in `else if` clause